### DISCUSSION TITLE
tools/wechat-mini-program-sourcemap

### DISCUSSION THREAD

**Discussion by wind-stone** [wind-stone/blog#306] [Announcements]
https://blog.windstone.cc/tools/wechat-mini-program-sourcemap

**Comment by 7935201** [comment]
source为啥都是空白的

**Comment by wind-stone** [comment]
> @7935201
> source为啥都是空白的

先确认下是否在上传小程序时开启了 ES6 转 ES5、代码压缩，只有这么设置才能生成正常的 Source Map，否则生成的 Source Map 就是空的，详见：https://blog.windstone.cc/articles/wechat-mini-program-sourcemap/

**Comment by Lgowen** [comment]
hello可以请教一下我uniapp编译后的源码包无法还原的问题吗
